go/iot/protocol/examples/arduino/protocol.ino

```diff
@@ -134,8 +148,12 @@ void readLine(char * buffer, int maxLength)
   buffer[idx] = 0;
 }
 
+response* parseResponse(char *response){
+  //return parseResponseJson(response);
```

Comment:
You can remove these legacy lines. You can revert them back via the history if they are needed for reference.